TrigScint/exampleConfigs/firmwareEx.py

```diff
@@ -0,0 +1,175 @@
+from LDMX.Biasing import filters
+
+from LDMX.Detectors.makePath import *
+from LDMX.SimCore import simcfg
+
+#pull in command line options
+nEle=4      # simulated beam electrons
+runNum=10
+version="ldmx-det-v14"
+outputNameString= "ldmxdetv14gap10mm_firmware.root" #sample identifier
+outDir= ""    #sample identifier
+
+#
+# Instantiate the simulator.
+#
+sim = simulator.simulator("test")
+
+#
+# Set the path to the detector to use (pulled from job config)
+#
+sim.setDetector( version, True )
+sim.scoringPlanes = makeScoringPlanesPath(version)
+
+outname=outputNameString #+".root"
+print("NAME = " + outname)
+
+#
+# Set run parameters. These are all pulled from the job config 
+#
+p.run = runNum
+p.maxEvents = 100
+nElectrons = nEle
+beamEnergy = 4.0;  #in GeV                                                                                                                                              
+
+sim.description = "Inclusive "+str(beamEnergy)+" GeV electron events, "+str(nElectrons)+"e"
+#sim.randomSeeds = [ SEED1 , SEED2 ]
+sim.beamSpotSmear = [20., 80., 0]
+
+
+mpgGen = generators.multi( "mgpGen" ) # this is the line that actually creates the generator                                                                  
+mpgGen.vertex = [ -44., 0., -880. ] # mm                                                                                                                              
+mpgGen.nParticles = nElectrons
+mpgGen.pdgID = 11
+mpgGen.enablePoisson = False #True                                                                                                                                      
+
+import math
+theta = math.radians(5.45)
+beamEnergyMeV=1000*beamEnergy
+px = beamEnergyMeV*math.sin(theta)
+py = 0.;
+pz= beamEnergyMeV*math.cos(theta)
+mpgGen.momentum = [ px, py, pz ]
+
+#
+# Set the multiparticle gun as generator
+#
+sim.generators = [ mpgGen ]
+         
+#reconstruction and vetoes 
+
+#Ecal and Hcal hardwired/geometry stuff
+#import LDMX.Ecal.EcalGeometry
+import LDMX.Ecal.ecal_hardcoded_conditions
+from LDMX.Ecal import EcalGeometry
+#egeom = EcalGeometry.EcalGeometryProvider.getInstance()
+#Hcal hardwired/geometry stuff
+from LDMX.Hcal import HcalGeometry
+import LDMX.Hcal.hcal_hardcoded_conditions
+#hgeom = HcalGeometry.HcalGeometryProvider.getInstance()
+
+
+from LDMX.Ecal import digi as eDigi
+from LDMX.Ecal import vetos
+from LDMX.Hcal import digi as hDigi
+from LDMX.Hcal import hcal
+
+from LDMX.Recon.simpleTrigger import TriggerProcessor
+
+from LDMX.TrigScint.trigScint import TrigScintDigiProducer
+from LDMX.TrigScint.trigScint import TrigScintClusterProducer
+from LDMX.TrigScint.trigScint import trigScintTrack
+from LDMX.TrigScint.trigScint import TrigScintFirmwareTracker
+
+if "v12" in version :
+     tsSimColls=[ "TriggerPadTagSimHits", "TriggerPadUpSimHits", "TriggerPadDnSimHits" ]
+else :
+     tsSimColls=[ "TriggerPad2SimHits", "TriggerPad3SimHits", "TriggerPad1SimHits" ]
```

Comment:
Is any of the v12 stuff still relevant?
```suggestion
tsSimColls=[ "TriggerPad2SimHits", "TriggerPad3SimHits", "TriggerPad1SimHits" ]
```

Reply:
No. I adopted an old working script here. Any naming convention works for me.